### PR TITLE
fixes broken faq links

### PR DESCRIPTION
**Pull request by gitbreaker222**
I hope its ok to directly present these small changes in a PR instead of creating an issue first :pray: 

## Before

Links lead to 404 (btw same as https://github.com/laurent22/joplin/blob/dev/CONTRIBUTING.md from the PR description template :stuck_out_tongue_winking_eye: )

https://joplinapp.org/faq/
https://joplinapp.org/faq/#desktop-application-will-not-launch-on-linux

## After

https://joplinapp.org/help/faq/
https://joplinapp.org/help/faq/#desktop-application-will-not-launch-on-linux

Feel free to close, if these places are wrong in any way :see_no_evil:

### CODE REVIEW

**Comment by gitbreaker222** [comment]
I have read the CLA Document and I hereby sign the CLA

File: readme/apps/terminal.md [hunk 1]
```diff
@@ -325,15 +325,15 @@ Possible keys/values:
...
+									https://joplinapp.org/help/faq/
 									Type: string.
 
 	sync.5.path                    Nextcloud WebDAV URL.
 									Attention: If you change this location,
 									make sure you copy all your content to it
 									before syncing, otherwise all files will be
 									removed! See the FAQ for more details:
-									https://joplinapp.org/faq/
+									https://joplinapp.org/help/faq/
```
Comment (laurent22) [inline readme/apps/terminal.md]: This one is already fixed, but in a different place (will be updated later)

File: Joplin_install_and_update.sh [hunk 2]
```diff
@@ -124,7 +124,7 @@ else
 fi
 if [[ $LIBFUSE == "" ]] ; then
   print "${COLOR_RED}Error: Can't get libfuse2 on system, please install libfuse2${COLOR_RESET}"
-  print "See https://joplinapp.org/faq/#desktop-application-will-not-launch-on-linux and https://github.com/AppImage/AppImageKit/wiki/FUSE for more information"
+  print "See https://joplinapp.org/help/faq/#desktop-application-will-not-launch-on-linux and https://github.com/AppImage/AppImageKit/wiki/FUSE for more information"
```
Comment (laurent22) [inline Joplin_install_and_update.sh:127]: This one is good

**Review by laurent22 — COMMENTED** [review]
Thanks, if you could remove the changes in terminal.md I can merge

**Pull request merged**